tests/Unit/Provider/Command/MessageSendTest.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\Mail\Tests\Unit\Provider\Command;
+
+use ChristophWurst\Nextcloud\Testing\TestCase;
+use OCA\Mail\Account;
+use OCA\Mail\Db\LocalAttachment;
+use OCA\Mail\Db\LocalMessage;
+use OCA\Mail\Db\MailAccount;
+use OCA\Mail\Provider\Command\MessageSend;
+use OCA\Mail\Service\AccountService;
+use OCA\Mail\Service\Attachment\AttachmentService;
+use OCA\Mail\Service\OutboxService;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\IConfig;
+use OCP\Mail\Provider\Address;
+use OCP\Mail\Provider\Attachment;
+use OCP\Mail\Provider\Exception\SendException;
+use OCP\Mail\Provider\Message;
+use PHPUnit\Framework\MockObject\MockObject;
+
+class MessageSendTest extends TestCase {
+	
+	private IConfig&MockObject $config;
+	private ITimeFactory&MockObject $time;
+	private AccountService&MockObject $accountService;
+	private OutboxService&MockObject $outboxService;
+	private AttachmentService&MockObject $attachmentService;
+	private $commandSend;
+
+	protected function setUp(): void {
+		parent::setUp();
+		// construct mock constructor parameters
+		$this->config = $this->createMock(IConfig::class);
+		$this->time = $this->createMock(ITimeFactory::class);
+		$this->accountService = $this->createMock(AccountService::class);
+		$this->outboxService = $this->createMock(OutboxService::class);
+		$this->attachmentService = $this->createMock(AttachmentService::class);
+		// construct test object
+		$this->commandSend = new MessageSend($this->config, $this->time, $this->accountService, $this->outboxService, $this->attachmentService);
+		// construct mail provider attachment
+		$this->mailAttachment = new Attachment(
+			'This is the contents of our plan',
+			'plan.txt',
+			'text/plain'
+		);
+		// construct mail provider message
+		$this->mailMessage = new Message();
```

Comment:
This is causing a deprecation warning because the class has no property  `mailMessage`. 

Not essential, not a blocker, but also not much to do: 

```
	private MessageSend $commandSend;
	private Attachment $mailAttachment;
	private Message $mailMessage;
	private Account $localAccount;
	private array $localMessageData;
	private array $localAttachmentData;
```

Reply:
Okay

Reply:
This is not resolved.

Reply:
Yes, I missed this one. Fixed.